schema.sql

```diff
@@ -80,4 +80,11 @@ CREATE TABLE visits (
     vet_id INT REFERENCES vets(id),
     visit_date DATE,
     PRIMARY KEY (animal_id, vet_id, visit_date)
-);
\ No newline at end of file
+);
+
+--Add an email column to your owners table
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX audit_visits ON visits (animal_id, vet_id);
+CREATE INDEX owner_email_desc ON owners(email DESC);
```

Comment:
Great job team 🤝

- [ ] As requested by the previous reviewer, kindly ensure the index for the email is added and is reflected in the output. To be sure that your query is using the index. There should be an index scan in the query output. This is not the case for your output. If you do this, you will notice a significant change in your execution time.
![index-email](https://github.com/Roland-Ntwali/Vet-clinic/assets/63917725/c28610e6-aaa1-4211-a92f-bcb2def861dc)